ansible/roles/sufia/defaults/main.yml

```diff
@@ -20,5 +20,9 @@ graylog_protocol: udp
 graylog_network_locality: WAN
 graylog_facility: "{{ project_name }}"
 graylog_verbosity: "info"
+# OSF API settings
+osf_client_id: ""
+osf_client_secret: ""
```

Comment:
Shouldn't this be `project_osf_client_secret`?

Reply:
Change made